monkestation/code/game/objects/items/anime.dm

```diff
@@ -108,15 +87,82 @@
 	M.equip_in_one_of_slots(new_item, slots , qdel_on_fail = TRUE)
 
 /obj/item/choice_beacon/anime/generate_display_names()
-	var/static/list/anime
+	var/list/anime
 	if(!anime)
 		anime = list()
-		var/list/templist = list(/obj/item/anime/cat, //Add to this list if you want your implant to be included in the trait
-								 /obj/item/anime/fox,
-								 /obj/item/anime/wolf,
-								 /obj/item/anime/shark
-								)
+		var/list/templist = list()
+		if(hacked_l == TRUE)
```

Comment:
Have the hacked choice beacon make the hacked variable change to true as it gives the user the item instead of having a full set of copied defines.

Reply:
When you say defines do you mean just the list in generate_display_name or do you mean all the anime/hacked children?

Reply:
I'm saying all of the children don't need to exist.
A hacked beacon can set the hacked variable in the item before giving it to the user.

Reply:
would it still be possible to change their name?

Reply:
Yes, do it just the same as you would modify any other variable.
You can have the kit create the item as a variable, then modify the name and hacked state of it.

Reply:
ah yeah, you can